parser/lexer.go

```diff
@@ -209,6 +188,47 @@ func isSymbol(r rune) bool {
 	return false
 }
 
+func isHorzWs(r rune) bool {
```

Comment:
nit: I wouldn't abbreviate it. I would go with`isHorizontalWhitespace`.

Reply:
Same with other functions with "Ws" in their names.

Reply:
Done.